README.md

```diff
@@ -1,3 +1,9 @@
+# Opendoor-related CHANGELOG
+
+* 1.5.4-opendoor.1
+    * Add a command entries to [helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/daemonset.yaml](https://github.com/opendoor-labs/splunk-connect-for-kubernetes/blob/master/helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/daemonset.yaml) and [helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-objects/templates/deployment.yaml](https://github.com/opendoor-labs/splunk-connect-for-kubernetes/blob/master/helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-objects/templates/deployment.yaml)
```

Comment:
need context on how did you derived the command
e.g. command value sourced from the official `Dockerfile` (and link the permanent link to the dockerfile line number)